### ISSUE TITLE
Right Joystick isn’t mapped correctly

### ISSUE THREAD

**Issue by Zamrod** [unknownskl/xbox-xcloud-client#221]
When using the right stick left and right on the stick seems to move up and down but up and down do nothing. 

Using the left trigger seems to move the right stick left or right depending on how far down you push it

**Comment by billbuc** [comment]
If you use the latest version, start the web server from the client and open http://local host:8080 from Edge the controls work fine

**Comment by rendar82** [comment]
I 2nd the comment from [Zamrod](https://github.com/Zamrod)

I am using this awesome program on an Arch Linux build of SteamOs 3.0 on he Steam Deck. I've been desperately searching for a working method of local xHome streaming for my Xbox Series X on the local LAN since, as the "official" Edge browser method from Microsoft is just for Gamepass Cloud streaming. This xbox-cloud-client is by far the most promising of all I've tried, and it would be absolutely perfect in every way for Stem Deck with the exception of the very strange gamepad mapping/controls issue.

When launching a console stream via the "Connect" button in the main UI (screen capture attached) everything seems to work great until you realize several game pad controls are not working or are mapped incorrectly. I have tried with the built in steam deck controller as well as Xbox one controller via Bluetooth, and the results were the same.

The view and menu buttons activate up/down on the d-pad, while the d-pad itself seems unmapped. The left joystick, ABXY face buttons and LB/RB bumpers appear to work correctly. The left and right trigger axis' seem to activate the right joystick x-axis, and he right joystick x-axis seems to activate right joystick Y-axis. Right joystick Y-axis seems unmapped, and it doesn't seem possible to activate the input for the triggers.

Per billbuc's comment, when starting the webui webserver and accessing it from a browser window, streaming the local console does appear to have correct/working gamepad mapping. I'm not exactly sure what the difference is, but I'm guessing the webserver is somehow using a different control scheme or different file for gamepad inputs.
The webserver doesnt work in SteamOS gaming mode and has other issues such as choppy sound, strange full screen behavior/controls among ohers.

The ability for user configuration of at least the gamepad or keyboard mapping or just proper default controls would make this the ultimate Xbox app for Linux.

Thank you fr the amazing work on this project!
